be1-go/hub/witness_test.go

```diff
@@ -0,0 +1,72 @@
+package hub
+
+import (
+	"bytes"
+	"context"
+	"encoding/base64"
+	"log"
+	"strings"
+	"student20_pop"
+	"sync"
+	"testing"
+)
+
+func CreateWitnessHub() Hub {
```

Comment:
Maybe you could return an error in case there the function fails to create a wintess hub: ` func CreateWitnessHub() (Hub, error)` but that is a detail

Reply:
a good idea, but when used the test checks that the witness is not nil and returns an error if it is